wordpress-mobile/WordPress-Android
# Refactor blaze packages

Part of #18532

This PR cleans up the blaze package structure within the ui package
- Renamed `blazewebview` package to `blazepromote` (including fragment/vm pairs)
- Moved all classes associated with promote into the `blazepromote`  package
- Moved the `blazeoverlay` and the new `blazepromote` package to the top level `blaze` package
- Removed the blaze `ui` package (now that it’s empty)
- Moved `blazeCampaigns` package under the `blaze` package
- Rename `blazeCampaigns` to `blazecampaign` to stay consistent with other package names (all lowercase)
- Kept shared classes under main blaze package directly (like BlazeActionEvent and BlazeWebClient)

To test:
There is nothing specific to test except to spin up the app and make sure blaze card is still shown and you can access the blaze promote feature.

## Regression Notes
1. Potential unintended areas of impact
Blaze entry point does not work

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:

UI Changes testing checklist: N/A - No UI changes
